### DISCUSSION TITLE
Why alpha versions for the spring-boot starter?

### DISCUSSION THREAD

**Discussion by rpgdev** [open-telemetry/opentelemetry-java-instrumentation#11650] [Q&A]
Hi, in here https://opentelemetry.io/docs/zero-code/java/spring-boot-starter/getting-started/ there are instructions to use the following dependency that's versioned as alpha despite being 2.X why?
![image](https://github.com/open-telemetry/opentelemetry-java/assets/141452028/1013c487-9980-40ed-8036-0d7c3b083fa9)

Also I can see there is a similarly named (but not alpha) artifact in [maven central](https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-instrumentation-bom), can I use one or the other? What's the difference?

**Comment by trask** [comment]
hi @rpgdev! it's because the OpenTelemetry Spring Boot Starter is not (quite) GA. we are currently targeting July release to mark it stable 🤞

**Comment by jkwatson** [comment]
moving this to the instrumentation repo...
